kotlinpoet/src/main/java/com/squareup/kotlinpoet/LambdaTypeName.kt

```diff
@@ -17,8 +17,11 @@ package com.squareup.kotlinpoet
 
 import kotlin.reflect.KClass
 
+@OptIn(ContextReceivers::class)
 public class LambdaTypeName private constructor(
   public val receiver: TypeName? = null,
+  @property:ContextReceivers
```

Comment:
Why the site target?

Reply:
IntelliJ gave me an error that the annotation wouldn't work on parameters, despite it also having the `VALUE_PARAMETER` target